PyInstaller/building/datastruct.py

```diff
@@ -50,7 +50,7 @@ def _normentry(self, entry):
             logger.info("TOC found a %s, not a tuple", entry)
             raise TypeError("Expected tuple, not %s." % type(entry).__name__)
         name, path, typecode = entry
-        if typecode in ["BINARY", "DATA"]:
+        if typecode in ["BINARY", "DATA"] and not name.endswith('.typelib'):
             # Normalize the case for binary files and data files only (to avoid duplicates
```

Comment:
This look **dangerous**. This will exclude **any** `typelib` module in **any** package. This should be much more specific or even better, handled in some hook or so.